### DISCUSSION TITLE
issues with forced encryption

### DISCUSSION THREAD

**Discussion by GirlGeek98** [arnoudkooi/SN-Utils#424] [General]
When you have encryption / crypto modules setup on your instance, using Ctrl-V will default to having the screen shot / attachment be encrypted.

![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/62078018/689c5778-a4ae-42eb-93ca-3f33ab0f9318)
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/62078018/db1518bd-8bb7-4c99-ae3b-ded6d94b49d0)
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/62078018/8e862c89-bf78-4533-8bfe-1df90d6f6d5e)

This could all be solved if ServiceNow would add a "None" option in the Crypto module picker, but who knows if that will every happen.  Any thoughts on a workaround?

**Comment by arnoudkooi** [comment]
Is this an enhancement request, or a problem with SN Utils?

**Reply by GirlGeek98** [reply]
Hmm... not sure.  Are the attachments pasting as encrypted an OOB design do you think?  If so, I guess I'm thinking a good enhancement would be to paste them un-encrypted.  If someone wanted to encrypt an attachment, they could go to the paperclip and manually pick an encryption context.

**Reply by arnoudkooi** [reply]
I do not have experience with this. So when manually uploading you can select none?

**Reply by GirlGeek98** [reply]
I had to adjust an OOB UI Page, but yes.  It almost seems in Utah if you have CLE encryption setup, it forces encryption on all your attachments.  This is a problem for us.

**Reply by arnoudkooi** [reply]
Have you raised a support case? If this is the designed behavior, I'm hesitant to change it.
